components/SpotCard.js

```diff
@@ -0,0 +1,64 @@
+import styled from "styled-components";
+
+export default function SpotCard({ name, address }) {
+  return (
+    <Card>
+      <Content>
+        <TextContainer>
+          <Name>{name}</Name>
+          <Address>{address}</Address>
+        </TextContainer>
+        <ImageContainer>
+          <Image
+            src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8e/Hauskatze_langhaar.jpg/600px-Hauskatze_langhaar.jpg"
+            alt="Spot Image"
+          />
+        </ImageContainer>
+      </Content>
+    </Card>
+  );
+}
+
+const Card = styled.div`
```

Comment:
You could consider making this an `article` or `section` tag instead of a `div`.

Reply:
Thank you!